### ISSUE TITLE
Decoders are difficult to understand, because online examples does not work

### ISSUE THREAD

**Issue by bobbysock** [wazuh/wazuh#26685]
|Wazuh version| Component | Action type |
|4.9.1| Decoders | Error |

The examples in the online documentation for Decoders does not work.
I am not going to include examples here, because if you copy paste from the docs, it will not work.

**Comment by jftuduri** [comment]
Hello @bobbysock ,

Thanks for reaching out! I reviewed the examples in the decoder documentation, and it appears there’s indeed an error in this section: [Sibling Decoders - Practical Example](https://documentation.wazuh.com/current/user-manual/ruleset/decoders/sibling-decoders.html#practical-example).

The log message provided in the example is:
```
2019/01/02 13:16:35 securityapp: INFO: srcuser="Bob" action="called" dstusr="Alice"
```
However, when this log format is tested with the `wazuh-logtest` utility, using the decoder from the documentation, the output is as follows:
```
Starting wazuh-logtest v4.9.1
Type one log per line

2019/01/02 13:16:35 securityapp: INFO: srcuser="Bob" action="called" dstusr="Alice"

**Phase 1: Completed pre-decoding.
	full event: '2019/01/02 13:16:35 securityapp: INFO: srcuser="Bob" action="called" dstusr="Alice"'

**Phase 2: Completed decoding.
	No decoder matched.
```
Which is not the expected output according to the documentation.

It seems that the example is using a wrong timestamp format which prevents it from matching correctly. Additionally, the hostname is missing, which may be another factor affecting the decoder’s performance.

To overcome this problem you can use a log sample like this:
```
Apr 12 14:31:38 hostname1 securityapp: INFO: srcuser="Bob" action="called" dstusr="Alice"
```

With this updated log format, the output is correct:
```
Starting wazuh-logtest v4.9.1
Type one log per line

Apr 12 14:31:38 hostname1 securityapp: INFO: srcuser="Bob" action="called" dstusr="Alice"


**Phase 1: Completed pre-decoding.
	full event: 'Apr 12 14:31:38 hostname1 securityapp: INFO: srcuser="Bob" action="called" dstusr="Alice"'
	timestamp: 'Apr 12 14:31:38'
	hostname: 'hostname1'
	program_name: 'securityapp'

**Phase 2: Completed decoding.
	name: 'securityapp'
	action: 'called'
	dstuser: 'Alice'
	srcuser: 'Bob'
	type: 'INFO'

```

In summary, the log format in the example at [Sibling Decoders - Practical Example](https://documentation.wazuh.com/current/user-manual/ruleset/decoders/sibling-decoders.html#practical-example) is currently incorrect. You should be able to proceed with the guide by using the log format shown above.

We will work on updating the documentation soon. Could you confirm if this was the issue you encountered? Did you find any other discrepancies in the documentation?

**Comment by bobbysock** [comment]
I can confirm that this was the issue. It works with the provided log sample.
